api/api.go

```diff
@@ -34,12 +40,34 @@ func Start(elrondFacade MainApiHandler) error {
 	if err != nil {
 		return err
 	}
-	registerRoutes(ws, elrondFacade)
+	registerRoutes(ws, elrondFacade, elrondFacade.PrometheusMonitoring())
```

Comment:
PrometheusMonitoring is accessible with elrondFacade which is already provided, so you could probably remove it as a parameter.

Reply:
done